editor.planx.uk/src/components/Feedback/index.tsx

```diff
@@ -138,14 +120,7 @@ const Feedback: React.FC = () => {
   }
 
   function ReportAnIssueTopBar(): FCReturn {
```

Comment:
The order of the conditions here have been reversed but the logic is still consistent - now we only show the "back" button if they've previously been on "triage", as opposed to not showing it if they were on "banner" (now removed as a view).